README.md

```diff
@@ -1,12 +1,222 @@
+Instead, multiple separate ResourceQuota objects would need to be created and managed
+individually for each namespace.
+
+Additionally, if a quota is set with `requests.cpu`, `limits.cpu`, `requests.memory`, or
+`limits.memory` boundaries in a namespace, then pods that don't set these resources are not
+allowed to be created. This essentially means that in such namespaces, best effort pods and some
+burstable pods are not allowed. The design decision which caused this limitation in Kubernetes 
+is admitted to be a mistake, but is still kept for backward compatibility reasons as mentioned 
+in the following
+[link](https://github.com/kubernetes/kubernetes/blob/c876b30c2b30c0355045d7548c22b6cd42ab58da/pkg/quota/v1/evaluator/core/pods.go#L89).
+
+## User Stories
+
+**User Story**: Managing VM Billing and Resource Quotas in a Multi-Tenant Cluster with KubeVirt
+
+KubeVirt is an operator that enables the deployment and management of virtual machines alongside
+containerized applications.
+
+As an administrator of a multi-tenant cluster running Virtual Machines with 
+[KubeVirt](https://github.com/kubevirt/kubevirt?tab=readme-ov-file#introduction, 
+I expect tenants to be billed only for the resources they explicitly request in the VM object. 
+However, this presents specific challenges with resource quotas:
+
+1. VM Live Migration: VM instances run inside pods. Occasionally, the pod associated with a VM needs to 
+be evicted from one node and rescheduled onto another. VMs have a large state that requires careful 
+handling during migration. To manage this, KubeVirt creates a new pod to transfer the VM state before 
+terminating the old one, which differs from how standard pods are moved. Running both pods simultaneously
+temporarily doubles the amount of resources the tenant is accounted for and can impact resource quotas, 
+potentially leading to migration issues and upgrade failures.
+2. In KubeVirt, each Virtual Machine operates within its own container. This means that the virtualization 
+overhead, such as the hypervisor, runs within each VM pod instead of centrally on the host, as in traditional 
+virtualization. This overhead is factored into quota calculations. In traditional virtualization platforms, 
+service providers typically base VM quotas solely on RAM and CPU threads, since users don't directly request 
+resources for this overhead or expect its effects. Moreover, the resource requirements for this overhead can 
+change between versions without users being notified.
+
+
+As a result, deploying KubeVirt in a multi-tenant cluster while restricting hardware consumption 
+from different entities is highly challenging. It requires significant resource allocation to 
+handle live migrations and manage VM infrastructure overhead. This also couples users to the 
+VM implementation, making them dependent on infrastructure overhead that can change unexpectedly.
+
+**User Story**: Creating non-guaranteed QoS Pods in a multi-tenant cluster
```

Comment:
Pods that are burstable can have requests and limits defined. How about
```suggestion
**User Story**: Creating Pods with lacking resource requests/limits in a multi-tenant cluster
```

Reply:
Done.